projects/igniteui-angular/src/lib/chips/chips-area.component.ts

```diff
@@ -24,13 +24,32 @@ import { CommonModule } from '@angular/common';
 import { CheckboxRequiredValidator, ControlValueAccessor, NG_VALIDATORS, NG_VALUE_ACCESSOR } from '@angular/forms';
 import { cloneArray } from '../core/utils';
 import { IgxRippleModule } from '../directives/ripple/ripple.directive';
-import { IgxChipComponent } from './chip.component';
+import {
+    IgxChipComponent,
+    IChipSelectEventArgs,
+    IChipKeyDownEventArgs,
+    IChipEnterDragAreaEventArgs,
+    IBaseChipEventArgs
+} from './chip.component';
 import { IgxDragDropModule } from '../directives/dragdrop/dragdrop.directive';
 import { IgxButtonModule } from '../directives/button/button.directive';
 import { IgxAvatarModule } from '../avatar/avatar.component';
 import { IgxIconModule } from '../icon';
 import { IgxConnectorDirective } from './connector.directive';
 
+export interface IBaseChipAreaEventArgs {
```

Comment:
The name of the component is `ChipsAreaComponent`, I think the names should be consistent so there is not confusion.
In this case would be `IBaseChipsAreaEventArgs`